python/lsst/afw/math/chebyshevBoundedField.cc

```diff
@@ -28,6 +28,10 @@
 #include "ndarray/pybind11.h"
 #include "ndarray/converter.h"
 
+#include "lsst/pex/config/pybind11.h"
```

Comment:
It would be nice to have a comment here to let future developers know this was included for LSST_DECLARE_CONTROL_FIELD, so they know where the macro originates